tests/integration/targets/prepare_lab/tasks/main.yml

```diff
@@ -1,3 +1,10 @@
+---
+- name: Fail If Running On Eco vCenter
+  ansible.builtin.fail:
+    msg: This test is very desctuctive and should only be run on Zuul!
+  when: >-
+    lookup('ansible.builtin.env', 'VCENTER_HOSTNAME') == "eco-vcenter-server.lab.eng.tlv2.redhat.com"
```

Comment:
let's put this domain (`eco-vcenter-server.lab.eng.tlv2.redhat.com`) inside a var so that if in the future the env will change we could change it directly from the var. wdyt?

Reply:
i wasnt really sure where this role gets its vars from, i think theres some static file on the zuul infra somewhere. but i added it to the `prepare_lab` role vars